src/Identity/Extensions.Core/src/PasswordValidator.cs

```diff
@@ -89,33 +90,43 @@ public virtual bool IsDigit(char c)
         }
 
         /// <summary>
-        /// Returns a flag indicating whether the supplied character is a lower case ASCII letter.
+        /// Returns a flag indicating whether the supplied character is a lower case unicode letter.
         /// </summary>
-        /// <param name="c">The character to check if it is a lower case ASCII letter.</param>
-        /// <returns>True if the character is a lower case ASCII letter, otherwise false.</returns>
+        /// <param name="c">The character to check if it is a lower case unicode letter.</param>
+        /// <returns>True if the character is a lower case unicode letter, otherwise false.</returns>
         public virtual bool IsLower(char c)
         {
-            return c >= 'a' && c <= 'z';
+            string s = c.ToString();
+            foreach (Rune r in s.EnumerateRunes())
+            {
+                if (Rune.IsLower(r)) return true;
+            }
+            return false;
         }
```

Comment:
I'm unsure if I implemented it correctly. Mentioning @GrabYourPitchforks for review.